packages/@amazeelabs/silverback-iframe/drupal/silverback_iframe/silverback_iframe.module

```diff
@@ -89,3 +89,37 @@ function silverback_iframe_entity_type_build(array &$entity_types) {
     $entity_types['webform_submission']->setFormClass('add', WebformSubmissionForm::class);
   }
 }
+
+/**
+ * Implements hook_webform_submission_form_alter().
+ */
+function silverback_iframe_webform_submission_form_alter(array &$form, \Drupal\Core\Form\FormStateInterface $form_state, $form_id) {
+  // Only proceed for webform submissions in iframe mode
+  if (!silverback_iframe_theme_enabled()) {
+    return null;
+  }
+
+  $form_object = $form_state->getFormObject();
+  if (!$form_object instanceof \Drupal\webform\WebformSubmissionForm) {
```

Comment:
`use Drupal\webform\WebformSubmissionForm`
and then
`instance of WebformSubmissionForm`